OpenRA.Mods.Common/Commands/ChatCommands.cs

```diff
@@ -36,9 +36,9 @@ public bool OnChat(string playername, string message)
 				var command = Commands.FirstOrDefault(x => x.Key == name);
 
 				if (command.Value != null)
-					command.Value.InvokeCommand(name.ToLowerInvariant(), message.Substring(1 + name.Length).Trim());
+					command.Value.InvokeCommand(name, message.Substring(1 + name.Length).Trim());
```

Comment:
This is referenced under "ChatCommands: Handle "/" correctly." but I haven't seen any explanation or discussion about why the current behaviour is wrong?  IMO there is no problem with supporting case insensitive commands.

Reply:
It is already to lowercase at https://github.com/LipkeGu/OpenRA/blob/b9d071b2e6c2a37c211cadc64618c894b5c5b153/OpenRA.Mods.Common/Commands/ChatCommands.cs#L35